<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerspectiveSettingsDialog</class>
 <widget class="QDialog" name="PerspectiveSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perspective settings</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Field of View - Vertical Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Aspect ratio:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>degrees</string>
   </property>
  </widget>
  <widget class="QPushButton" name="useImageAspectRatioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Use image aspect ratio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fovYEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="aspectRatioEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fovXEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Field of View - Horizontal X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>degrees</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateFOVButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate FOV</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
